sdks/python/apache_beam/ml/transforms/tft_test.py

```diff
@@ -576,6 +574,17 @@ def setUp(self) -> None:
   def tearDown(self):
     shutil.rmtree(self.artifact_location)
 
+  def validate_count_per_key(self):
+    import os
+    key_vocab_location = os.path.join(
+        self.artifact_location, 'transform_fn/assets/key_vocab')
+    with open(key_vocab_location, 'r'):
```

Comment:
```suggestion
    with open(key_vocab_location, 'r') as f:
```

Reply:
Ah this is to be removed since I moved this fn internal to the test. Thanks for catching. I will remove this.